proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -26,6 +26,8 @@ The proposed mechanism is detailed below:
   	2. construct the `helpPrompt` parameter using the data from the list SDL Core internally created.
   	3. then send these parameters to the HMI via the `SetGlobalProperties` RPC.
 5. If after SDL Core sends the `SetGlobalProperties` RPC, applications sends further AddCommmand/DeleteCommand requests then SDL Core shall send `SetGlobalProperties` with the updated parameters.(The full list of parameters needs to be sent and not just the recently added one).
+The limitation for processing AddCommand and providing updated values of "vrHelp" and "helpPrompt" to HMI till mobile application sends SetGlobalProperties request is 30.
+Update values of "vrHelp" and "helpPrompt" via TTS UI.SetGlobalProperties to HMI only by receving AddCoomad with CommandType = Command. AddCommand requests related to choice set must not trigger the update of "vrHelp" and "helpPrompt" values.
```

Comment:
Please fix the following typos:
- "receving" should be "receiving"
- "AddCoomad" should be "AddCommand"